main.tf

```diff
@@ -62,6 +62,7 @@ resource "google_compute_instance" "vm_instance" {
   metadata = {
     enable-oslogin = (var.os_login_enabled ? "TRUE" : "FALSE") # see https://cloud.google.com/compute/docs/instances/managing-instance-access#enable_oslogin
     windows-keys   = ""                                        # Placeholder to ignore changes. See https://www.terraform.io/docs/configuration/resources.html#ignore_changes
+    attached_disk  = null
```

Comment:
Kindly add the helper comment like given for windows-keys. It allows us developers to know `null` is not the intended value of this field but rather just a placeholder value - so in future it helps us avoid looking into this for any potential trouble-shooting efforts

Reply:
Done